ports/rp2/mpconfigport.h

```diff
@@ -61,6 +61,7 @@
 #define MICROPY_OPT_COMPUTED_GOTO               (1)
 
 // Python internal features
+#define MICROPY_TRACKED_ALLOC                   (MICROPY_SSL_MBEDTLS)
```

Comment:
Motion for `MICROPY_TRACKED_ALLOC` to be enabled generally. We have a few modules that it would be useful for.

Reply:
It adds a small bit of overhead (an extra word for the GC to check) if not used.  Maybe better to wrap this in `#ifndef MICROPY_TRACKED_ALLOC` ..., and then a board can optionally enable it?